pkg/kapis/gitops/v1alpha1/registery.go

```diff
@@ -35,6 +35,7 @@ func AddToContainer(container *restful.Container, options *common.Options, argoO
 		runtime.NewWebServiceWithoutGroup(v1alpha1.GroupVersion),
 	}
 	for _, service := range services {
+		RegisterRoutes(service, options, argoOption)
```

Comment:
these routes are used by all gitops enginess, so if the argoOption is needed for this?